server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/GroupItem.java

```diff
@@ -31,8 +31,8 @@
 public class GroupItem extends StructuredVariable {
 
   public GroupItem(
-      int levelNumber, String name, Locality definition, Variable parent) {
-    super(levelNumber, name, definition, parent);
+      int levelNumber, String name, Locality definition, Boolean global, Variable parent) {
```

Comment:
Why obj ref. Boolean, will you use it for 'null' case? in `AbstractVariable ` it defined as `protected final boolean global;`